app/src/main/java/io/pslab/activity/RoboticArmActivity.java

```diff
@@ -226,11 +232,14 @@ public void onStopTrackingTouch(SeekArc seekArc) {
         seekArc3.setOnSeekArcChangeListener(new SeekArc.OnSeekArcChangeListener() {
             @Override
             public void onProgressChanged(SeekArc seekArc, int i, boolean b) {
+                degreeText3.requestFocus();
```

Comment:
If we're making the cursor invisible, let's remove this request focus part and keep setcursorvisible false outside if else statement.